frontend/src/Components/homeView/HomeViewPage.tsx

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from "react";
+import { connect } from "react-redux";
+import { Link } from "react-router-dom";
+import {
+  Props as DefaultProperties,
+  defaultMapStateToProps,
+  defaultMapDispatchToProps,
+} from "../../Interfaces/DefaultConnections";
+import SessionListView from '../sessionList/SessionListView';
+import LocalStorageService from '../../Services/LocalStorageService';
+
+const HomeViewPage = (props: DefaultProperties) => {
+  let [localStateArray, setlocalStateArray] = useState(LocalStorageService.GetSessions());
+
+  const deleteSession = (sessionId: any) => {
```

Comment:
`sessionId: any` should instead be the specific type of number or string (whichever it is supposed to be)